odl/util/utility.py

```diff
@@ -163,6 +185,145 @@ def is_complex_floating_dtype(dtype):
     return np.issubsctype(dtype, np.complexfloating)
 
 
+def default_dtype(impl, field):
```

Comment:
I mean the code [here](https://github.com/odlgroup/odl/pull/120/files#diff-c53e5a0e4ff98988cce249116ccaaabcR882)